src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private static final Integer RANDOM_BOUND = 100;
```

Comment:
static fields should be at the beginning of the class
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents